trySwift/RootTabBar/RootTabBarController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  RootTabBarController.swift
+//  trySwift
+//
+//  Created by Sash Zats on 3/2/18.
+//  Copyright © 2018 NatashaTheRobot. All rights reserved.
+//
+
+import UIKit
+
+class RootTabBarController: UITabBarController, UITabBarControllerDelegate {
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    self.delegate = self
+  }
+
+  func tabBarController(_ tabBarController: UITabBarController, didSelect viewController: UIViewController) {
+    if let splitViewController = viewController as? UISplitViewController {
```

Comment:
Maybe `guard let`s and early exits might make this function a bit more readable?